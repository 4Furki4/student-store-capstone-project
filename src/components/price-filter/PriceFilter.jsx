const PriceFilter = () => {
    return (
        <div className='flex justify-center'>
            <div className='xs:scale-[70%] sm:w-80 sm:scale-[80%] md:scale-100 xl:w-96'>
                <div className='input-group relative flex w-full flex-wrap items-stretch'>
                    <input
                        type='search'
                        className='form-control relative m-0 block w-full min-w-0 flex-auto rounded border border-solid border-gray-300 bg-white bg-clip-padding px-3 py-1.5 text-base font-normal text-gray-700 transition ease-in-out focus:border-purple focus:bg-white focus:text-gray-700 focus:outline-none'
                        placeholder='Max price'
                        aria-label='Search'
                        aria-describedby='button-addon2'
                    />
                </div>
            </div>
        </div>
    );
};

export default PriceFilter;
