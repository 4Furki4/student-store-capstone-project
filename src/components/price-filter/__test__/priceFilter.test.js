import renderer from "react-test-renderer";

import PriceFilter from "../PriceFilter";

it("renders correctly", () => {
    const tree = renderer.create(<PriceFilter />).toJSON();
    expect(tree).toMatchSnapshot();
});
