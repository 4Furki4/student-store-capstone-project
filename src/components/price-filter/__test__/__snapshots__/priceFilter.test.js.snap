// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex justify-center"
>
  <div
    className="xs:scale-[70%] sm:w-80 sm:scale-[80%] md:scale-100 xl:w-96"
  >
    <div
      className="input-group relative flex w-full flex-wrap items-stretch xs:pb-2 lg:pb-0"
    >
      <input
        aria-describedby="button-addon2"
        aria-label="Search"
        className="form-control relative m-0 block w-[50%] min-w-0 flex-auto rounded-2xl border border-solid border-gray-300 bg-white bg-clip-padding px-3 py-1.5 text-base font-normal text-gray-700 transition ease-in-out focus:border-purple focus:bg-white focus:text-gray-700 focus:outline-none"
        placeholder="Min price"
        type="search"
      />
      <input
        aria-describedby="button-addon2"
        aria-label="Search"
        className="form-control relative m-0 block w-[50%] min-w-0 flex-auto rounded-2xl border border-solid border-gray-300 bg-white bg-clip-padding px-3 py-1.5 text-base font-normal text-gray-700 transition ease-in-out focus:border-purple focus:bg-white focus:text-gray-700 focus:outline-none"
        placeholder="Max price"
        type="search"
      />
    </div>
  </div>
</div>
`;
